mkdocs.yml

```diff
@@ -116,66 +114,37 @@ nav:
-                      - microservices/device/services/device-onvif-camera/supplementary-info/api-analytic-support.md
-                      - microservices/device/services/device-onvif-camera/supplementary-info/api-event-handling.md
-                      - microservices/device/services/device-onvif-camera/supplementary-info/api-usage-user-handling.md
-                  - Utility:
-                      - microservices/device/services/device-onvif-camera/supplementary-info/test-with-postman.md
-                      - microservices/device/services/device-onvif-camera/supplementary-info/utility-scripts.md
-                  - Miscellaneous:
-                      - microservices/device/services/device-onvif-camera/supplementary-info/control-plane-events.md
-                      - microservices/device/services/device-onvif-camera/supplementary-info/getting-started-with-docker-security.md
-                      - microservices/device/services/device-onvif-camera/supplementary-info/ws-discovery.md
-                      - microservices/device/services/device-onvif-camera/supplementary-info/onvif-footnotes.md
-          - Device REST: microservices/device/services/device-rest.md
-          - Device RFID LLRP: microservices/device/services/device-rfid-llrp.md
-          - Device SNMP: microservices/device/services/device-snmp.md
-          - Device UART: microservices/device/services/device-uart.md
-          - Device USB Camera:
-              - microservices/device/services/device-usb-camera/General.md
-              - Walkthrough:
-                  - microservices/device/services/device-usb-camera/walkthrough/setup.md
-                  - microservices/device/services/device-usb-camera/walkthrough/deployment.md
-                  - microservices/device/services/device-usb-camera/walkthrough/general-usage.md
-                  - microservices/device/services/device-usb-camera/walkthrough/custom-build.md
-              - Supplementary Information:
-                  - microservices/device/services/device-usb-camera/supplementary-info/USB-protocol.md
-                  - microservices/device/services/device-usb-camera/supplementary-info/discovery.md
-                  - microservices/device/services/device-usb-camera/supplementary-info/advanced-options.md
-          - Device Virtual: microservices/device/services/device-virtual/Ch-VirtualDevice.md
-          - Remote Deployment:
-              - Non-Secure Mode: microservices/device/remote/Ch-RemoteNonSecure.md
-              - Secure Mode: microservices/device/remote/Ch-RemoteSecure.md
+        - Device Service: microservices/device/DeviceService.md
+        - Getting Started: microservices/device/GettingStarted.md
+        - Configuration: microservices/device/Configuration.md
+        - API Reference: microservices/device/ApiReference.md
+        - Device Service SDK:
+            - Purpose: microservices/device/sdk/Purpose.md
+            - Getting Started: 
+                -  C Device SDK:
```

Comment:
I don't think we need the extra level here since the is only one page for each. It is needed for APIs since each has multiple pages.

Reply:
updated